app/assets/scripts/reducers/reducer.js

```diff
@@ -376,6 +376,11 @@ const VProMMsWayBbox = function (state = VProMMsWayBboxDefaultState, action) {
       state.fetching = false;
       state.fetched = true;
       state.bbox = action.json;
+      break;
+    case actions.REMOVE_VPROMMS_BBOX:
+      state = _.cloneDeep(state);
+      state = VProMMsWayBboxDefaultState;
```

Comment:
There's no need to clone the state here. You can just return the default one since no modifications are done.